### ISSUE TITLE
Exception [WinError 995] The I/O operation has been aborted

### ISSUE THREAD

**Issue by jquast** [jquast/blessed#121]
<img width="944" alt="Screen Shot 2020-01-15 at 10 29 25 PM" src="https://user-images.githubusercontent.com/972584/72499385-89ac5600-37e6-11ea-937c-8f57c23bd9d6.png">

When initializing a terminal using IPython, we get this frightening error,

```
Unhandled exception in event loop:
  File "c:\python38\lib\asyncio\proactor_events.py", line 768, in _loop_self_reading
    f.result()  # may raise
  File "c:\python38\lib\asyncio\windows_events.py", line 808, in _poll
    value = callback(transferred, key, ov)
  File "c:\python38\lib\asyncio\windows_events.py", line 457, in finish_recv
    raise ConnectionResetError(*exc.args)

Exception [WinError 995] The I/O operation has been aborted because of either a thread exit or an application request
Press ENTER to continue...
```

**Comment by jquast** [comment]
This may be https://github.com/prompt-toolkit/python-prompt-toolkit/issues/1023#issue-534396318

**Comment by avylove** [comment]
This seems to be a [bug](https://bugs.python.org/issue39010) in Python 3.8. I can't reliably produce it with Blessed to test the workarounds. It would also be nice to know which part of `__init__()`  triggers it, but I haven't been able to figure that out yet.

**Comment by avylove** [comment]
Looks like this is a [bug](https://github.com/ipython/ipython/issues/12049) in iPython/prompt-toolkit >= 3.  Easy to reproduce just holding down enter in the iPython console. I haven't been able to reproduce with Blessed in standard Python. I think we can close.

**Comment by jdpipe** [comment]
I was able to reproduce this error on a brand-new clean install of python-3.8.2-amd64.exe in Win 10 (with no custom options selected). I opened IPython by invoking this 'launch.py' script by name from the cmd prompt:
```python
import sys,os.path as o,subprocess as c;
c.call(o.join(o.dirname(sys.executable),'Scripts','ipython.exe'))
```
and then I simply typed 'enter' into the prompt several times. It would show this error ever few times.

The error was solved by forcing:
```bash
py -m pip install -U prompt-toolkit~=2.0
```
